src/components/ClosePositionModal/index.js

```diff
@@ -88,6 +88,13 @@ export const ClosePositionModal = ({isModalVisible,onCancel,position,chainId, ..
       }   
     },[position]);
 
+    useEffect(() => {
+      if(position && percentage){
+        const amount = position.position * parseFloat(percentage) / 100
+        setTokenAmount(amount)
+      }   
+    },[percentage,tokenAmount,position]);
```

Comment:
variables and functions used within the scope should go into dependency array. [Ref](https://stackoverflow.com/questions/71814755/use-case-for-passing-function-as-an-dependency-in-useeffect-in-react)